### PR TITLE
feat: Pass logged in agent context to dashboard app

### PR DESCRIPTION
**Pull request by nithindavid**
Fixes https://github.com/chatwoot/chatwoot/issues/5898

### CODE REVIEW

**Comment by nithindavid** [comment]
@fayazara Thanks for pointing out this. I have made the changes.

**Review by fayazara — APPROVED** [review]
Screenshot2022-12-08 at 00 49 01
Tested locally - working as expected

File: app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue [hunk 1]
```diff
@@ -46,11 +46,21 @@ export default {
...
     },
     contactId() {
       return this.currentChat?.meta?.sender?.id;
     },
+    currentAgent() {
+      const {
+        id,
+        name,
+        email,
+        phone_number: phone,
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue:60]:
```suggestion
```

Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue:60]: There is no phone number in the agent model.

**Pull request merged**